### DISCUSSION TITLE
I paid for teams but still can't use codespaces

### DISCUSSION THREAD

**Discussion by DiamondYuan** [community/community#5414] [Codespaces] [answered]
what should i do ?

**Comment by kiriDevs** [comment]
Re-Read Codespaces FAQs to see if you missed something, if you didn't I guess you should contact support.

**Comment by tanmayeekamath** [comment, marked as answer]
Hey @DiamondYuan, please reach out to codespaces@github.com with your GH Team name and we can get you enabled. :)
